app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    humans = [Person(person["name"], person["age"]) for person in people]
+    for human in humans:
+        for person in people:
+            if (
+                "wife" in person
+                and person["wife"] is not None
+                and human.name == person["name"]
+            ):
+                human.wife = Person.people[person["wife"]]
+            elif "husband" in person and person["husband"] is not None:
```

Comment:
You could write this check in one line, e.g.:
```suggestion
            elif person.get("husband"):
```
You could make your previous check shorter as well with the same approach.